tests/Swagger/Serializer/DocumentationNormalizerTest.php

```diff
@@ -1111,6 +1111,8 @@ public function testNoOperations()
      */
     public function testMethodNotSupported()
```

Comment:
We may want to use custom methods (standard ones like PATCH, but also custom ones like BAN or PURGE).